docs/architecture/server/index.md

```diff
@@ -0,0 +1,119 @@
+---
+sidebar_position: 30
+---
+
+# Server Architecture
```

Comment:
ℹ️ Casing of titles is inconsistent. Not something I am hung up for now, but I do want to fix this en masse.

Reply:
True. I've made all sub-headings sentence case (because they mostly were already) but left this top-level heading using title case (which seems to be consistent across pages). I guess that makes it consistently inconsistent?